dist/gcc-compatible/Hacl_Hash_Blake2b.c

```diff
@@ -679,46 +679,138 @@ void Hacl_Hash_Blake2b_finish(uint32_t nn, uint8_t *output, uint64_t *hash)
   Lib_Memzero0_memzero(b, 64U, uint8_t, void *);
 }
 
-/**
-  State allocation function when there is no key
-*/
-Hacl_Hash_Blake2b_state_t *Hacl_Hash_Blake2b_malloc(void)
+uint8_t *FStar_Pervasives_Native_snd__uint32_t__uint8_t_(K___uint32_t__uint8_t_ x)
```

Comment:
Nit: Could we use a struct instead of a pair to avoid these names ?

Reply:
as discussed in PR #930, I cannot use structs because it doesn't monomorphize correctly, and I can't hide the names because of an incorrect handling of internal headers and CAbstractStruct in krml -- to be fixed in a followup